x/config/doc.go

```diff
@@ -179,35 +186,131 @@
 // 	  oneway:
 // 	    # ...
 //
+// Peer Configuration
+//
+// Transports that support peer management and selection through YARPC accept
+// some additional keys in their outbound configuration.
+//
+// An explicit peer may be specified for a supported transport by using the
+// `peer` option.
+//
+// 	keyvalue:
+// 	  tchannel:
+// 	    peer: 127.0.0.1:4040
+//
+// All requests made to this outbound will be made through this peer.
+//
+// If a peer list was registered with the system, the name of the peer list
+// may be used to specify a more complex peer selection and load balancing
+// strategy.
+//
+// 	keyvalue:
+// 	  http:
+// 	    url: https://host/yarpc
+// 	    round-robin:
+// 	      peers:
+// 	        - 127.0.0.1:8080
+// 	        - 127.0.0.1:8081
+// 	        - 127.0.0.1:8082
+//
+// In the example above, the system will round-robin the requests between the
+// different addresses. The specified URL will be used as a template for the
+// HTTP requests made to these hosts.
```

Comment:
Perhaps clarify that this templating is specific to the HTTP transport, and that other transports may not support the same thing?